src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -260,6 +287,40 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
         return columns;
     }
 
+    /*******************************************************************************************************
+    * @description gets updated amount total and row count for the batch
+    * @param batchId: ID of the NPSP_Data_Import_Batch__c
+    * @return DataImportModel updated model with total amount and number of rows
+    */
+    private static DataImportModel getUpdatedTotals(Id batchId) {
```

Comment:
I find this method name confusing given that it returns an intact Model. `getUpdatedModel` maybe?

Reply:
Yes, this felt awkward to me too. What I *really* wanted to do was make a new model that just returns the totals + a single DI record, but wasn't sure if that was making the controller too bulky. But that's really what I'm looking for when I'm calling this, and just using the model as a way to pass those values back together. I need the totals + DI record for run dry run on a new record or edited record, and then just need the totals for when delete is called.